ops/terraform/config/dev.config

```diff
@@ -1,4 +1,4 @@
-    storage_account_name = "tfstaterv2024"
-    container_name       = "rv-tfstate"
-    key                  = "dev.terraform.tfstate"
-    use_oidc             = true
\ No newline at end of file
+storage_account_name = "tfstaterv2024"
```

Comment:
Assuming these are for each of the different dev environments?

Reply:
that correct! so with TF you can store the configuration state files either locally on our computers or a in a central location. these config files basically create and update these files from a global AZ Resource Group in the same account subscription. the benefit of seperating the out into individual files like this is so we can use the `-backend-config=config/${{ inputs.deploy-env }}.config` flag in the `terraform` cli. We are no using that here in a reusable GHA: https://github.com/CDCgov/ReportVision/blob/fcdfeeb5dd0026075aa3e589432f0ccc9bbe6c14/.github/actions/tf-setup/action.yml#L53